### PR TITLE
default tranformation --> no transform

### PR DESCRIPTION
**Pull request by jhamman**

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Notes:
 | lon_name              | string  | Yes       | n/a     |                                                                                          |
 | time_name             | string  | Yes       | n/a     |                                                                                          |
 | nfiles                | integer | Yes       | -1      |                                                                                          |
-| transformations       | integer | No        | 2       | no transform = 1, quantile mapping = 2, log transform = 3, cube root = 4, fifth root = 5 |
```
Comment (gutmann) [inline docs/config.md]: off by one error again, kNO_TRANFORM=0, QM=1,...

Comment (jhamman) [inline docs/config.md]: yep, all good here now.  Just needed to merge in #19.

File: docs/config.md [hunk 2]
```diff
@@ -76,8 +76,8 @@ Notes:
 | lon_name              | string  | Yes       | n/a     |                                                                                          |
 | time_name             | string  | Yes       | n/a     |                                                                                          |
 | nfiles                | integer | Yes       | -1      |                                                                                          |
-| transformations       | integer | No        | 1       | no transform = 1, quantile mapping = 2, log transform = 3, cube root = 4, fifth root = 5 |
-| input_transformations | integer | No        | 1       | no transform = 1, quantile mapping = 2, log transform = 3, cube root = 4, fifth root = 5 |
+| transformations       | integer | No        | 0       | no transform = 0, quantile mapping = 1, log transform = 2, cube root = 3, fifth root = 4 |
+| input_transformations | integer | No        | 1       | no transform = 0, quantile mapping = 1, log transform = 2, cube root = 3, fifth root = 4 |
```
Comment (gutmann) [inline docs/config.md]: also, input_transform default is 0 (no transform)

Comment (jhamman) [inline docs/config.md]: You are right. That should have actually made it into #19 but I'll sneak it in here.

**Pull request merged**